daemon/resources/razer.conf

```diff
@@ -18,3 +18,9 @@ mouse_battery_notifier_freq = 600
 
 # Apply effects saved to disk when daemon starts
 restore_persistence = False
+
+# Keyboard battery notifier
+keyboard_battery_notifier = True
+
+# Keyboard battery notification frequency [s] (0 to disable)
+keyboard_battery_notifier_freq = 600
```

Comment:
I don't think it makes much sense to keep keyboard and mouse separated. If it's not too much work please merge them, remove the `mouse_` prefix and keep a migration path so that `mouse_battery_notifier` and `mouse_battery_notifier_freq` don't stop working with this change.
Otherwise we can also keep it like this for now and merge the items them later.

Reply:
Done